lib/broadside/ecs/ecs_deploy.rb

```diff
@@ -67,7 +67,7 @@ def rollback(options = {})
     end
 
     def scale(options = {})
-      info "Rescaling #{family} with scale=#{@scale}..."
+      info "Rescaling #{family} with scale=#{options[:scale]}..."
```

Comment:
may not be guaranteed to be `options[:scale]` if it's not passed in as I believe it's optional (https://github.com/lumoslabs/broadside/blob/169a2a3fe50734347da4417c7678914c8d9d5837/lib/broadside/ecs/ecs_deploy.rb#L144)